### DISCUSSION TITLE
When I set-retention for a partitioned topic by admin cli, whether it will take effect in every partitions ？

### DISCUSSION THREAD

**Discussion by philixtian** [apache/pulsar#18168] [Q&A] [answered]
As the title says, will all partitions get the same retention polices?

**Comment by Jason918** [comment, marked as answer]
Yes, topic partitions share all the same policies.

**Reply by philixtian** [reply]
Thanks~

**Reply by philixtian** [reply]
For example, set-retention to a partitioned-topics with 1T 3day, is that means every single partition of this topic got 1T , or all partitions share 1T?
@Jason918

**Reply by Jason918** [reply]
"every single partition of this topic got 1T"
